tests/integration/examples/example_tests/proxy_connectivity_check.rs

```diff
@@ -47,11 +44,7 @@ async fn test_http_client_over_socks5_proxy_connect() {
         "local servers up and running",
     );
 
-    // TODO: once we have socks5 support in Easy http web client
-    // we can probably simplify this by using the interactive runner client
-    // instead of having to do this manually...
-
-    let client = HttpConnector::new(Socks5ProxyConnector::required(TcpConnector::new()));
+    let client = EasyHttpWebClient::default();
```

Comment:
What I meant is that in all these socks5 tests we shouldn't even have a need to create our own Client, right? We should be able to just run `runner.get(...).send(...)` no? Just need to make sure that our default runner client is making use of the auto proxy support, which I guess it already is anyway?